src/main/java/CompanyUtils.java

```diff
@@ -0,0 +1,39 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class CompanyUtils {
+
+    public static double[] statistics(Employee[] employees) {
+        double[] statistics = new double[6];
+        statistics[0] = StatisticsCalculator.avgSalary(employees);
+        statistics[1] = StatisticsCalculator.minSalary(employees);
+        statistics[2] = StatisticsCalculator.maxSalary(employees);
+        statistics[3] = StatisticsCalculator.countItEmployee(employees);
+        statistics[4] = StatisticsCalculator.countSupportEmployee(employees);
+        statistics[5] = StatisticsCalculator.countManagementEmployee(employees);
+        return statistics;
+    }
```

Comment:
Zamiast zwracać tablicę `double[]`, lepiej stwórz dodatkową klasę np. Statistics. W przypadku tablicy musimy wierzyć w to, że się nie pomyliliśmy w kolejności, a w przypadku klasy mamy jasno zdefiniowane nazwy pól, więc o pomyłkę dużo trudniej